src/master/develop/deleting.md

```diff
@@ -0,0 +1,65 @@
+# Deleting
+
+It's possible to achieve deletion by using the `Delete` function. It provides logical deletion, it means that it is not phisically deleted from db, but it's not possible to query it anymore after deletion.
+
+
+:::: tabs
+
+::: tab Go
+```go
+    tx, err = client.Set(ctx, []byte("1,2,3"), []byte("3,2,1"))
+	if  err != nil {
+		log.Fatal(err)
+	}
+
+	fmt.Printf("Successfully committed tx %d\n", tx.Id)
+
+	entry, err := client.Get(ctx, []byte("1,2,3"))
+	if  err != nil {
+		log.Fatal(err)
+	}
+
+	fmt.Printf("Successfully retrieved entry: %v\n", entry)
+
+	_, err = client.Delete(ctx, []byte("1,2,3"))
+	if  err != nil {
+		log.Fatal(err)
+	}
+
+	//the following will raise an error with key not found
+	_, err = client.Get(ctx, []byte("1,2,3"))
+	if  err != nil {
```

Comment:
probably compare that the error is not nil and contains the text of "key not found"

Reply:
But the objective of the example is to show the error "key not found"